domain/src/main/kotlin/com/example/domain/usecase/implement/WeatherUseCase.kt

```diff
@@ -0,0 +1,18 @@
+package com.example.domain.usecase.implement
+
+import com.example.domain.WeatherRepositoryInterface
+import com.example.domain.entities.ResponseWeatherInfo
+import com.example.domain.usecase.WeatherUseCaseInterface
+import kotlinx.coroutines.flow.Flow
+import org.koin.core.KoinComponent
+
+class WeatherUseCase(
+    private val weatherRepositoryInterface: WeatherRepositoryInterface
```

Comment:
Nitpick: rename the variable to just weatherRepository

Reply:
Done